src/server.js

```diff
@@ -17,6 +18,7 @@ server.use(express.json())
 // routers
 server.use('/users', usersRouter);
 server.use('/logbooks', logbooksRouter);
+server.use('/tablesISR', tablesISRRouter);
```

Comment:
@AldaMayorgaDev yo dejaría el endpoint todo en minusculas

```suggestion
server.use('/tables-isr', tablesISRRouter);
```